Tests/Indicators/RelativeVigorIndexTests.cs

```diff
@@ -41,7 +41,7 @@ public void ComparesWithExternalDataRviSignal()
                 TestFileName,
                 "RVI_S",
                 (ind, expected) => Assert.AreEqual(
-                    ind.IsReady ? expected : 0,
```

Comment:
I wasn't a huge fan about needing to tie the main indicator's IsReady to the signal (as one can still use the RVI without the signal for the 3 extra terms it needs to warm up). I also didn't exactly like my original implementation for dealing with the issue in this older version of the test. Instead, removing the extra data from spy_rvi was a safer equivalent to `ind.IsReady ? expected : 0`. 

One can still check signal readiness by `Signal.IsReady` but it will be independent of "ind.IsReady".